web/html/src/manager/virtualization/ListTab.js

```diff
@@ -85,13 +86,14 @@ export function ListTab(props: Props) {
     const action = actionData.find(item => item.type === actionType);
     if (action) {
       return (
-        <ModalButton
+        <Button
           className="btn-default btn-sm"
           title={action.name}
           icon={action.icon}
-          target={`${actionType}-modal`}
-          item={row}
-          onClick={setSelected}
+          handler={() => {
+            setSelected(row);
+            setOpenedModals(Object.assign({}, openedModals, {[`${actionType}-modal`]: true}));
```

Comment:
the verbosity here comes from the fact that the `ListTab` component has a number of modals. Otherwise just a simple state variable like `modalOpened` would do

Reply:
Ah okay, I missed that bit and that does explain it well. In that case, a comment would help a lot please.